### PR TITLE
Move gems to gemfile versioned

### PR DESCRIPTION
**Pull request by MalloZup**
- [ ] merge this before : https://github.com/SUSE/spacewalk/pull/2916

### CODE REVIEW

**Comment by Bischoff** [comment]
LGTM

(why removing "unzip" ? Is that safe ?)

**Comment by mbologna** [comment]
Please fix title for PR and address @Bischoff comment. Apart from that, looks good.
Good idea to use Gemfile!

**Comment by MalloZup** [comment]
ok thx. the wip is intentional, because i'm testing it right now, i will remove the WIP when i am fineshed.

For @Bischoff  comments, we used ```unzip``` in the past when we weren't using the state ```git``` , so now is just a leftover that we can remove.

File: salt/controller/init.sls [hunk 1]
```diff
@@ -43,28 +43,31 @@ cucumber_requisites:
       - git-core
       - wget
       - aaa_base-extras
-      - unzip
+      - zlib-devel
```
Comment (MalloZup) [inline salt/controller/init.sls]: needed for `nokogiri` gem

**Comment by moio** [comment]
Please remove the WIP as soon as you are ready

**Comment by MalloZup** [comment]
@moio  yop, i am on it, i will remove the WIP once i finishing testing with the gems. :guitar: :rocket:

**Pull request merged**